vsg/__main__.py

```diff
@@ -42,6 +44,42 @@ def parse_command_line_arguments():
         return parser.parse_args()
 
 
+def get_predefined_styles():
+    '''
+    Reads all predefined styles and returns a list of names.
+
+    Parameters : None
+
+    Returns : (list of strings)
+    '''
+    lReturn = []
+    sStylePath = os.path.join(os.path.dirname(__file__), 'styles')
+    lStyles = os.listdir(sStylePath)
+    for sStyle in lStyles:
+        if sStyle.endswith('.yaml'):
+            with open(os.path.join(sStylePath, sStyle)) as yaml_file:
+                tempConfiguration = yaml.full_load(yaml_file)
+            lReturn.append(tempConfiguration['name'])
+    return lReturn
+
+
+def read_predefined_style(commandLineArguments):
```

Comment:
The parameter of this function should rather be `name`, not `commandLineArguments`.
Having to all command line arguments indicates that something is probably wrong with the logic.

Reply:
I see what you are saying.  It came about because I was calling `open_configuration_file`, which takes commandLineArguments.  I will work on improving the function.

Reply:
I have pushed an update for this comment.